### DISCUSSION TITLE
Missing configuration for delete post

### DISCUSSION THREAD

**Discussion by fulljackz** [posativ/isso#791] [Q&A]
When I play with mail notification and want to delete some comments, I can't delete them because of a 404 when i click on : http://mywebsite.com/2/delete/Mg.CsMu2w.ipTgXNmWSdsqZAZEsSQAl4eQxQ8 for example.

Is there anywhere some nginx config file to deal with these 404 errors ?

Thanks.

**Comment by ix5** [comment]
@fulljackz would you kindly verify this is still an issue?

**Comment by fulljackz** [comment]
Hi @ix5 ,
I don't really known, but as far as I do not use isso anymore, you can close I guess :smile:
